documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_3_Governance_Perspective/4_3_X_Goverance_for_Vocabularies.md

```diff
@@ -0,0 +1,9 @@
+### Governance for Vocabularies ###
+
+- shared vocabularies need a governance
+- Problem description: reliability of vocabularies, quality assurance
+- BOMOS Model as good practice?
+- Mapping of Standards (written paper) to machine readable models leaves some  degrees of freedom
+- 
```

Comment:
Add good practice: DILIGENT framework, naming conventions, expressivity. Automated checking: what can a consumer expects, documentation, self-explanatory, does it work in an automated environment. 
Link to DILIGENT